extra/pattern/equilateral.js

```diff
@@ -0,0 +1,14 @@
+function Equilateral(NoOfrow){
+    for(var i=1;i<=NoOfrow;i++){
+        for(var x=NoOfrow-1;x>=i;x--){
+            process.stdout.write(" ");
+        }
+        for(var j=1;j<=i;j++){
+            process.stdout.write("* ");
+
+        }
+        console.log();
+    }
+}
+var y=process.argv[2];
```

Comment:
Use const, let instead of var

Reply:
As discuss, also correct the PR subject line as per given format
<type>: <Subject of Ticket> (#<issue_number>)
i.e. feat: Sample Task (#38840)